[Bug]: satisfies operator breaks ts_project builds

### What happened?

Is it safe to upgrade @npm_typescript to 4.9.* https://github.com/aspect-build/rules_ts/blob/9e03fd92eebec869b9c092272b421550a3c1c1f5/WORKSPACE#LL20-L20C22?

Is there a workaround that users can define in their workspaces in the meantime?

`bazel build //typescript/...`

`typescript/core/theme/Fonts.ts: error TS2304: Cannot find name 'satisfies'.`

### Version

`bazel run -- @npm_typescript//:tsc --version`

`Version 4.8.2`

### How to reproduce

```shell
Use satisfies operator in ts code
```


### Any other information?

_No response_

### Fund our work

- [ ] Sponsor our open source work by donating a [bug bounty](https://opencollective.com/aspect-build/)

--- a/e2e/workspace/WORKSPACE
+++ b/e2e/workspace/WORKSPACE
@@ -14,9 +14,16 @@ local_repository(
 # you should fetch it *before* calling this.
 # Alternatively, you can skip calling this function, so long as you've
 # already fetched all the dependencies.
-load("@aspect_rules_ts//ts:repositories.bzl", "LATEST_VERSION", "rules_ts_dependencies")
+load("@aspect_rules_ts//ts:repositories.bzl", "rules_ts_dependencies")
 
-rules_ts_dependencies(ts_version = LATEST_VERSION)
+rules_ts_dependencies(
+    # This keeps the TypeScript version in-sync with the editor, which is typically best.
+    ts_version_from = "//:package.json",
+
+    # Alternatively, you could pick a specific version, or use
+    # load("@aspect_rules_ts//ts:repositories.bzl", "LATEST_VERSION")
+    # ts_version = LATEST_VERSION
+)
 
 # Fetch and register node, if you haven't already
 load("@rules_nodejs//nodejs:repositories.bzl", "DEFAULT_NODE_VERSION", "nodejs_register_toolchains")

--- a/e2e/workspace/package.json
+++ b/e2e/workspace/package.json
@@ -1,0 +1,5 @@
+{
+    "devDependencies": {
+        "typescript": "4.8.4"
+    }
+}

